app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -97,12 +97,13 @@ public function register (Request $request)
     */
     public function confirmEmail ($token)
     {
-        // a dynamic / magic method:
-        $user = User::whereToken($token)->firstOrFail()->confirmEmail();
-
-        session()->flash('emailconfirmed', 'Your email has been confirmed. You may now Log in.');
-
-        return redirect('/');
+        // a dynamic / magic method
+        // bool
+        $verified = User::whereToken($token)->firstOrFail()->confirmEmail();
+        $auth = false;
+        $user = null;
+        if ($auth) $user = Auth::user();
+        return view('root', compact('auth', 'user', 'verified'));
```

Comment:
$auth and $user here will always be false and null, as the user need to verify their email to log in

We also need a space above every return method